alphadia/workflow/peptidecentric.py

```diff
@@ -444,89 +405,205 @@ def calibration(self):
             )
             return
 
-        self.start_of_calibration()
-        for current_epoch in range(self.config["calibration"]["max_epochs"]):
-            if self.check_epoch_conditions():
-                pass
+        if self.config["search"]["target_ms2_tolerance"] > 0:
```

Comment:
Very clean and understandable code!
Can you move this code up to 486 into a separate function which returns the optimizer list?